include/votca/xtp/gw.h

```diff
@@ -85,6 +87,7 @@ class GW {
 
  private:
   Index _qptotal;
+  Index _mixing_order;
```

Comment:
why add it in the options and as a member variable?

Reply:
options alone suffices, indeed